app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/Registration.scala

```diff
@@ -113,6 +113,9 @@ case class Registration(
 
   val isFirstGroupMember = groupDetail.exists(_.members.isEmpty)
 
+  def populateBusinessRegisteredAddress(): Registration =
```

Comment:
Does this belong here? It feels like it should be in `RegistrationBuilder`? There seems to be some blurring of responsibilities between `RegistrationBuilder` and the `Registration` object.

Reply:
scratch that - builder is test-only